ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala

```diff
@@ -290,10 +290,10 @@ object HttpServiceTestFixture extends LazyLogging with Assertions with Inside {
       }
   }
 
-  def postJsonStringRequest(uri: Uri, jsonString: String, headers: List[HttpHeader])(
+  def postJsonStringRequestString(uri: Uri, jsonString: String, headers: List[HttpHeader])(
```

Comment:
```suggestion
  def postJsonStringRequestEncoded(uri: Uri, jsonString: String, headers: List[HttpHeader])(
```

🤷‍♂️

Reply:
Why `Encoded`? The `String` at the end is for the return type

Reply:
Because what is returned is the encoded form, rather than the decoded form. (The type is in the type 🙂 )

Reply:
Fair enough, naming things is hard :grin: